assets/multiqc_config.yml

```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/rnaseq" target="_blank">nf-core/rnaseq</a>
+  This report has been generated by the <a href="https://github.com/nf-core/rnaseq/3.12.0" target="_blank">nf-core/rnaseq</a>
```

Comment:
```suggestion
  This report has been generated by the <a href="https://github.com/nf-core/rnaseq/3.13.0dev" target="_blank">nf-core/rnaseq</a>
```

Reply:
version should be the same everywhere, no?

Reply:
ok, you already had an issue for it: https://github.com/nf-core/tools/issues/2361

Reply:
Yep, I have removed it for now until that issue is fixed:
https://github.com/nf-core/rnaseq/pull/1053/commits/7a5c4bb3966852c79a25aea14a1e10bb4e884d5d